javascript/ql/src/Security/CWE-178/CaseSensitiveMiddlewarePath.qhelp

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Using a case-sensitive regular expression path in a middleware route enables an attacker to bypass that middleware
+when accessing an endpoint with a case-insensitive path.
```

Comment:
I think we need to explain that explicit string routes are treated case-insensitively.